mechanicalsoup/browser.py

```diff
@@ -139,8 +139,16 @@ def post(self, *args, **kwargs):
         Browser.add_soup(response, self.soup_config)
         return response
 
-    def _request(self, form, url=None, **kwargs):
-        """Extract input data from the form to pass to a Requests session."""
+    @classmethod
+    def _get_request_kwargs(cls, method, url, **kwargs):
+        # This method exists to raise a TypeError when a method or url is
+        # specified in the kwargs.
```

Comment:
Shouldn't this comment be a docstring?

Reply:
Actually, this is my first time writing a docstring.

I'll be referring to other methods to write docstrings. Please let me know if you have any concerns.